### ISSUE TITLE
Backdoor:Win32/Bladabindi!ml found in pytorch_model.bin

### ISSUE THREAD

**Issue by patrickmros** [oobabooga/text-generation-webui#456]
Windows defender claims to have found a backdoor in C:\Users\patri\.cache\huggingface\hub\models--Salesforce--blip-image-captioning-base\snapshots\9f3084c0600e544506a2d414647492a036fb1aa0\pytorch_model.bin

False Alarm or something to worry about?

**Comment by the-real-molskin** [comment]
same.
Detected: Backdoor:Win32/Bladabindi!ml
Status: Removed
file: C:\Users\molsk\invokeai\models\diffusers\models--stabilityai--stable-diffusion-2-1\snapshots\f7f33030acc57428be85fbec092c37a78231d75a\text_encoder\pytorch_model.bin

**Comment by Arargd** [comment]
I looked up the name, and it would appear that a lot of people are getting it for all sorts of random things. At least a few reddit posts and some other forum post. I'm imagining its a false positive and potentially something wrong with their system.

**Comment by patrickmros** [comment]
> something like VirusTotal

Yeah that was the first thing I've tried. But the file size is to large. I will install a second virus scanner to check the file.

**Comment by BetaDoggo** [comment]
Classic windows defender. I've gotten similar alerts for other pytorch models. Almost definitely a false positive.

**Comment by gwyong** [comment]
I also got this error when using Salesforce-blip-image-captioning. What should I do?
